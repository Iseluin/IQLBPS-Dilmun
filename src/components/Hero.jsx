
function Hero() {
  return (
    <div className='backdrop-blur-md bg-white/30 h-[100vh] cont text-center text-background'>
      <div className='backdrop-blur-md w-full h-full  flex flex-col justify-center text-center'>
        <div className=" text-bg-black sm:mx-0 mx-4">
          <h1 className=' text-background text-7xl font-bold  '>
            Welcome to Dilumn
          </h1>
          <p className='text-background text-xl my-4'>Through Dilumn You Can Get the Best Quality Products </p>
          <div className="btn__container">
            <button type='button' className="bg-secondary hover:bg-tertiary text-white font-bold py-2 px-4 rounded-full">
              Get Started
            </button>
          </div>
        </div>


      </div>
    </div>

  )
}

export default Hero;