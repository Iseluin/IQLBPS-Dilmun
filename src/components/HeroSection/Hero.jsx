function Hero() {
  return (
    <div className="backdrop-blur-md bg-white/30 h-[100vh] bg-hero-bg bg-no-repeat text-center text-background bg-cover">
      <div className="backdrop-blur-md w-full h-full  flex flex-col justify-center text-center">
        <div className=" text-bg-black md:mx-0 mx-4">
          <h1 className=" text-background text-7xl font-bold  ">
            Welcome to Dilmun
          </h1>
          <p className="text-background text-xl my-4">
            Your Way to Get Quality Products!
          </p>
          <div className="btn__container">
            <button
              type="button"
              className="bg-secondary hover:bg-tertiary text-white font-bold py-2 px-4 rounded-full"
            >
              Sign Up
            </button>
          </div>
        </div>
      </div>
    </div>
  );
}

export default Hero;
