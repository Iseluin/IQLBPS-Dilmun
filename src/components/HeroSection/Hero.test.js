import React from "react";
import { render } from "@testing-library/react";
import Hero from ".";


it('render correctly', () => {
    const { queryByPlaceholderText, getByText, queryByAltText } = render( < Hero / > )
    expect(getByText('Welcome to Dilmun')).toBeTruthy()
    expect(getByText('Your Way to Get Quality Products!')).toBeTruthy()

})