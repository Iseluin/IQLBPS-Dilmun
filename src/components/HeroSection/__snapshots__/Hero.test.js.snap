// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly: 

          <h1 className=' text-background text-7xl font-bold  '>
            Welcome to Dilmun
          </h1>
          <p className='text-background text-xl my-4'>Your Way to Get Quality Products!</p>
         
            <button type='button' className="bg-secondary hover:bg-tertiary text-white font-bold py-2 px-4 rounded-full">
            Sign Up
            </button>
    `;